custom_model_runner/datarobot_drum/drum/language_predictors/base_language_predictor.py

```diff
@@ -105,13 +107,24 @@ def configure(self, params):
         self._code_dir = params["__custom_model_path__"]
         self._params = params
 
+        if to_bool(params.get("allow_dr_api_access")):
```

Comment:
Looks like this was moved from `PythonPredictor` to `BaseLanguagePredictor`

Reply:
Yes